src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,11 +59,13 @@
 
     /**
      * Test body.
+     * @todo: #120 inline constant to test(s)
```

Comment:
@westonized maybe we could link to http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html so that new person will understand idea quickly?